x-pack/platform/plugins/shared/fleet/server/services/package_policy.ts

```diff
@@ -841,12 +844,13 @@ class PackagePolicyClientImpl implements PackagePolicyClient {
     );
 
     const savedObjectType = await getPackagePolicySavedObjectType();
+    const packagePolicyLatestRevisionKuery = `(${savedObjectType}.attributes.latest_revision:true OR NOT ${savedObjectType}.attributes.latest_revision:*)`;
```

Comment:
we could add a saved object migration to backfill existing SOs with `latest_revision:true`, so the kuery wouldn't have to include the not exists filter